hr_payroll_commission/models/hr_payslip.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# © 2016 Coninckx David (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields, api
+
+
+class HrPayslip(models.Model):
+    _inherit = 'hr.payslip'
+
+    # ---------- Fields management
+
+    invoices = fields.One2many('account.invoice', 'slip_id',
+                               string='Invoices')
+    move_lines = fields.One2many('account.move.line', 'slip_id',
+                                 string='Journal Items')
+
+    # ---------- Utilities
+
+    def _detach_invoices_from_payslip(self):
```

Comment:
Why a new methos, isn't it easier to have self.invoices.unlink() in compute_sheet...